config/default/jenkins-release.yaml

```diff
@@ -16,7 +15,7 @@ jenkins:
       namespaceLabels:
         name: "release"
   controller:
-    image: jenkinsciinfra/jenkins-lts
+    image: 0.0.8-2.277.1
```

Comment:
There is an issue with this updateCli pipeline @garethjevans : the tag ins inserted into the `image` key